### ISSUE TITLE
se-resnet bockbone

### ISSUE THREAD

**Issue by gongzaizhou** [facebookresearch/maskrcnn-benchmark#714]
## ❓ Questions and Help
When I used se-resnet50 backbone, the accuracy dropped by 3 points , I used pretrained form https://github.com/Cadene/pretrained-models.pytorch

**Comment by SkeletonOne** [comment]
@gongzaizhou Hi, I'm trying to use se-resnet50 as well but the loss run into nan. I just changed the WEIGHT in the config file origin RetinaNet-50-FPN-1x to se-resnet50.pth. Do I need more changes to use this backbone?

**Comment by Dorozhko-Anton** [comment]
1. Did you trained faster_rcnn with se-resnet50 backbone starting from weights from https://github.com/Cadene/pretrained-models.pytorch?   
2. In comparison to what model the accuracy dropped by 3 points ?
